app/bundles/EmailBundle/Tests/Model/EmailModelFunctionalTest.php

```diff
@@ -164,4 +166,141 @@ public function testNotOverwriteChildrenTranslationEmailAfterSaveParent(): void
        self::assertSame($customHtmlParent, $parentEmail->getCustomHtml());
        self::assertSame($customHtmlChildren, $childrenEmail->getCustomHtml());
    }
+
+    public function testGetDeliveredCount(): void
+    {
+        $contact1 = $this->createContact('john@example.com');
+        $contact2 = $this->createContact('paul@example.com');
+
+        $this->em->flush();
+        $email   = $this->createEmailWithParams(
+            'Email A',
+            'Email A Subject',
+            'list',
+            'beefree-empty',
+            'Test html'
+        );
+        $this->em->flush();
+
+        $this->createEmailStat($contact1, $email);
+        $this->createEmailStat($contact2, $email);
+        $email->setSentCount(2);
+        $this->em->persist($email);
+        $this->em->flush();
+
+        $this->createDoNotContact($contact2, $email, DoNotContact::BOUNCED);
+        $this->em->flush();
+
+        /** @var EmailModel $emailModel */
+        $emailModel = self::$container->get('mautic.email.model.email');
```

Comment:
Please call the `getEmailDeliveredCountAction` here instead of just the model method. This way you'll get the action tested too and we'll know that that endpoint is actually working for users. It's a pity to test just a part of this PR.

Reply:
Right, done